component/core/src/main/java/org/exoplatform/social/core/service/LinkProvider.java

```diff
@@ -199,6 +199,9 @@ public static String getProfileLink(String username, String portalOwner) {
                .append("external: '")
                .append(identity.getProfile().getProperty(Profile.EXTERNAL) != null && StringUtils.equals("true", String.valueOf(identity.getProfile().getProperty(Profile.EXTERNAL))))
                .append("',")
+               .append("enabled: '")
+               .append(identity.isEnable() && !identity.isDeleted() && StringUtils.equals("true", String.valueOf(identity.isEnable())))
```

Comment:
why parsing boolean to String again ?

```suggestion
               .append(identity.isEnable() && !identity.isDeleted())
```

Reply:
removed!